### PR TITLE
(DOCSP-11907): add dgram to partially supported module for mdb realm functions

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-11097

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/docsp-11907-dgram-node-function-dependency/functions/built-in-module-support.html#dgram

### CODE REVIEW

**Review by cbush — APPROVED** [review]
Was it decided that we shouldn't mention that dgram is actually a forked version?

File: source/functions/built-in-module-support.txt [hunk 1]
```diff
@@ -94,6 +94,27 @@ process
 - :nodejs:`version <docs/v10.18.1/api/process.html#process_process_version>`
 - :nodejs:`versions <docs/v10.18.1/api/process.html#process_process_versions>`
 
+dgram
```
Comment (cbush) [inline source/functions/built-in-module-support.txt:97]: Could you please move this section to be in alphabetical order with the others?

File: source/functions/built-in-module-support.txt [hunk 2]
```diff
@@ -94,6 +94,27 @@ process
...
+~~~~~
+{+service-short+} supports the following ``dgram`` APIs:
+
+- :nodejs:`createSocket <api/dgram.html#dgram_dgram_createsocket_options_callback>`
+- :nodejs:`addMembership <api/dgram.html#dgram_socket_addmembership_multicastaddress_multicastinterface>`
```
Comment (cbush) [inline source/functions/built-in-module-support.txt:102]: I think these should be updated as follows:
```suggestion
- :nodejs:`socket.addMembership <api/dgram.html#dgram_socket_addmembership_multicastaddress_multicastinterface>`
```

File: source/functions/built-in-module-support.txt [hunk 3]
```diff
@@ -94,6 +94,27 @@ process
...
+- :nodejs:`address <api/dgram.html#dgram_socket_address>`
+- :nodejs:`bind <api/dgram.html#dgram_socket_bind_port_address_callback>`
+- :nodejs:`close <api/dgram.html#dgram_socket_close_callback>`
+- :nodejs:`dropMembership <api/dgram.html#dgram_socket_dropmembership_multicastaddress_multicastinterface>`
+- :nodejs:`send <api/dgram.html#dgram_socket_send_msg_offset_length_port_address_callback>`
+- :nodejs:`setBroadcast <api/dgram.html#dgram_socket_setbroadcast_flag>`
+- :nodejs:`setMulticastLoopback <api/dgram.html#dgram_socket_setmulticastloopback_flag>`
+- :nodejs:`setMulticastTTL <api/dgram.html#dgram_socket_setmulticastttl_ttl>`
```
Comment (cbush) [inline source/functions/built-in-module-support.txt:110]: ...to here:
```suggestion
- :nodejs:`socket.setMulticastTTL <api/dgram.html#dgram_socket_setmulticastttl_ttl>`
```

File: source/functions/built-in-module-support.txt (hunk 1 above)
Comment (mongomoe) [reply source/functions/built-in-module-support.txt:97]: 👍

File: source/functions/built-in-module-support.txt (hunk 2 above)
Comment (mongomoe) [reply source/functions/built-in-module-support.txt]: fixed

File: source/functions/built-in-module-support.txt (hunk 3 above)
Comment (mongomoe) [reply source/functions/built-in-module-support.txt]: fixed

**Comment by mongomoe** [comment]
@cbush to address your top-level comment ("Was it decided that we shouldn't mention that dgram is actually a forked version?")

Since the other modules in this page that are also forked versions don't mention that they are forks, Tim Sedgwick (Realm Cloud) suggested to leave it w/o the mention that it's a fork.

**Pull request merged**